applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+improving the user experience.
+Both individuals and enterprises looking for improved liquidity in their blockchain assets can leverage
+NFTAA for better asset management.
+Additionally, researchers and academics interested in exploring new blockchain innovations can study and expand upon the
+unique concepts introduced by NFTAA.
+
+The NFTAA project addresses several critical needs within the blockchain ecosystem.
+Utilizing NFTAA provides more methods for managing and delegating staking rights without exposing private keys.
+This approach significantly reduces the risk of unauthorized access and enhances overall security.
+Furthermore, NFTAA allows staked assets to remain liquid, enabling users to trade, lend, or use them as collateral even
+while they are staked.
+This flexibility is crucial for users who need access to their assets without compromising their staking benefits.
+The project also potentially leverages Polkadot's ability to connect multiple blockchains, ensuring seamless asset transfers and
+interactions across different networks.
+This interoperability is essential for creating a cohesive and efficient blockchain ecosystem.
+Moreover, NFTAA empowers users by allowing them to delegate staking rights, transfer ownership of staked assets, and
+interact with multiple blockchain networks through a single NFTAA, thereby enhancing user control and simplifying asset
+management.
+
+The needs addressed by the NFTAA project were identified through a comprehensive approach.
+A literature review of existing research on staking mechanisms, blockchain security, and asset management highlighted
+the limitations and potential improvements in current systems.
+Case studies examining current issues faced by stakers and asset managers in blockchain networks like Ethereum and
+Polkadot provided practical insights into the challenges and opportunities for innovation.
+Additionally, research on the drawbacks of existing staking mechanisms and the potential benefits of using NFTs for
+account management provided a solid foundation for the project's conceptual framework.
+
+There are similar projects in the Polkadot universe:
+
+- Lido: A liquid staking protocol that allows users to stake assets and receive derivatives representing the staked
+  amount.
+- Bifrost: Provides liquidity for staked assets through a voucher system and supports cross-chain staking.
+- Acala: Offers a similar staking mechanism with its LDOT token, providing immediate liquidity and flexibility in
+  staking.
+
+Differences and Distinctions:
+
+- NFTAA vs. Lido/Bifrost/Acala: Unlike these projects, NFTAA focuses on using NFTs as proxy accounts for staking, which
+  allows for a more flexible and secure delegation of staking rights and ownership. NFTAA also leverages the composability
+  of NFTs, enabling them to hold other assets and act as accounts within the blockchain ecosystem.
```

Comment:
How is this more secure? Because you don't rely on the business logic of a party like Acala but instead you rely on the business logic of yours in the form of this very pallet?

Reply:
Safer from the user's point of view that this way we can bypass various time locks. So we don't expose our self to the danger of illiquidity. Or, as we mentioned before, we take more flexible as more secure.

We don't question Acala's shared security model as it is inherited from the relay chain, but in our opinion, the pallet is more testable than the whole parachain or at least quicker and easier testable. Therefore we think that the easiness to test the pallet makes it at the end more robust (and maybe also secure).